.pullapprove.yml

```diff
@@ -1,7 +1,7 @@
 approve_by_comment: true
-approve_regex: '^(Approved|:shipit:|:\+1:)'
-reject_regex: '^(Rejected|:-1:)'
-reset_on_push: false
+approve_regex: '^(Approved|:shipit:|:\+1:|Engage)'
+reject_regex: '^(Rejected|:-1:|Borg)'
+reset_on_push: true
```

Comment:
Now that Stale check is gone, we don't need to merge base back in. Does possibly allow for some merge conflicts, but until PA updates and allows for certain commits to be bypassed then this will do.